src/quicktype-core/support/Strings.ts

```diff
@@ -519,18 +519,24 @@ export function makeNameStyle(
     let firstWordAcronymStyle: WordStyle;
     let restAcronymStyle: WordStyle;
 
-    if (namingStyle === "pascal" || namingStyle === "camel") {
+    if (namingStyle === "pascal" || namingStyle === "camel" || namingStyle === "pascal-upper-acronyms" || namingStyle === "camel-upper-acronyms") {
         separator = "";
-        restWordStyle = firstUpperWordStyle;
-        restAcronymStyle = allUpperWordStyle;
+        if (namingStyle.includes("upper-acronyms")) {
```

Comment:
I think there's some compatibility issue for why we can't use `.includes`.  It's not in IE or something.  I also dislike depending on the exact value of the string - it makes search-and-replace harder, for example.  Please just do a comparison with both values.